packages/utils/src/stacktrace.ts

```diff
@@ -64,28 +71,31 @@ export function stripSentryFramesAndReverse(stack: StackFrame[]): StackFrame[] {
 
   let localStack = stack;
 
-  const firstFrameFunction = localStack[0].function || '';
-  const lastFrameFunction = localStack[localStack.length - 1].function || '';
-
-  // If stack starts with one of our API calls, remove it (starts, meaning it's the top of the stack - aka last call)
-  if (firstFrameFunction.indexOf('captureMessage') !== -1 || firstFrameFunction.indexOf('captureException') !== -1) {
-    localStack = localStack.slice(1);
+  if (stack.length > STACKTRACE_LIMIT) {
```

Comment:
This slice could in theory be removed as the stack parser only ever returns a max of 50 frames, but since it is a separate exported fn, I did not check who/where it is called so it could still be possible that someone is passing > 50 frames as the argument.